jormungandr/src/explorer/graphql/scalars.rs

```diff
@@ -92,3 +117,33 @@ impl From<chain_time::TimeOffsetSeconds> for TimeOffsetSeconds {
         TimeOffsetSeconds(format!("{}", u64::from(time)))
     }
 }
+
+impl From<u32> for TransactionCount {
+    fn from(n: u32) -> TransactionCount {
+        TransactionCount(format!("{}", u32::from(n)))
+    }
+}
+
+impl From<u32> for IndexCursor {
```

Comment:
If this is not checked in any way on conversion, why the hassle with newtypes?

Reply:
I can't expose a `u32` directly to graphql. It only has i32 and floats, so I need to pass it as a string. I'm just using this to have a name (different from String) for it in the graphql schema.  Do you think the `From` implementation is unnecessary?

Reply:
I get the purpose now, thanks. It's a good practice actually.